spring-cloud-servicecomb-discovery/src/main/java/org.springframework.cloud.servicecomb.discovery/client/ServiceCombClient.java

```diff
@@ -226,6 +227,32 @@ public String registerInstance(MicroserviceInstance microserviceInstance)
     return null;
   }
 
+  /**
+   *  deRegister Instance
+   * @param serviceId
+   * @param instanceId
+   * @throws ServiceCombException
+   */
+  public boolean deRegisterInstance(String serviceId, String instanceId)
+      throws ServiceCombException {
+    Response response = null;
+    try {
+      String formatUrl = buildURI("/registry/microservices/" + serviceId + "/instances/" + instanceId);
+      response = httpTransport
+          .sendDeleteRequest(formatUrl);
+      if (response.getStatusCode() == HttpStatus.SC_OK) {
```

Comment:
if response is null , will throw ‘ build url failed ’ RemoteOperationException，but it should throw ‘ deRegister failed’

Reply:
Response will never be empty.
public Response execute(HttpUriRequest httpRequest) throws RemoteServerUnavailableException {
    Response resp = new Response();
URISyntaxException catch buildURI 
buildURI(String path) throws URISyntaxException